target/linux/mpc85xx/files/arch/powerpc/platforms/85xx/wg-t30w.c

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+/*
+ * WatchGuard T30-W board Board Setup
+ *
+ * Copyright (C) 2021 Damien Mascord <tusker@tusker.org>
+ *
+ * Based on:
+ *   mpc85xx_rdb.c:
+ *      MPC85xx RDB Board Setup
+ *      Copyright 2013 Freescale Semiconductor Inc.
+ *
+ * This program is free software; you can redistribute  it and/or modify it
+ * under  the terms of  the GNU General  Public License as published by the
+ * Free Software Foundation;  either version 2 of the  License, or (at your
+ * option) any later version.
+ */
+
+#include <linux/stddef.h>
+#include <linux/kernel.h>
+#include <linux/delay.h>
+#include <linux/interrupt.h>
+#include <linux/of_platform.h>
+
+#include <asm/time.h>
+#include <asm/machdep.h>
+#include <asm/pci-bridge.h>
+#include <mm/mmu_decl.h>
+#include <asm/prom.h>
+#include <asm/udbg.h>
+#include <asm/mpic.h>
+
+#include <sysdev/fsl_soc.h>
+#include <sysdev/fsl_pci.h>
+
+#include "mpc85xx.h"
+
+void __init wg_t30w_pic_init(void)
+{
+	struct mpic *mpic;
+
+	mpic = mpic_alloc(NULL, 0,
+	  MPIC_BIG_ENDIAN |
+	  MPIC_SINGLE_DEST_CPU,
+	  0, 256, " OpenPIC  ");
+
+	BUG_ON(mpic == NULL);
+	mpic_init(mpic);
+}
+
+/*
+ * Setup the architecture
+ */
+static void __init wg_t30w_setup_arch(void)
+{
+	if (ppc_md.progress)
+		ppc_md.progress("wg_t30w_setup_arch()", 0);
+
+	//mpc85xx_smp_init();
```

Comment:
we fixed the SMP issue with dd7d4703e9de73153bd239afcf67c77cdb7f7cf8, right?

Reply:
Let me test it with master, and then remove the comment if it works properly

Reply:
@chunkeey P1011 is single-core anyways :^)